elements/pfe-clipboard/src/pfe-clipboard.scss

```diff
@@ -5,7 +5,6 @@ $LOCAL: clipboard;
 
 // This variable is global so that helper functions can reference it
 $LOCAL-VARIABLES: (
-  FontSize: 1rem,
```

Comment:
@starryeyez024 Should we be deprecating functionality?  What if someone is using this already?

Reply:
this doesn't actually do anything now. previously it was responsible for creating the fallback value in the clipboard CSS, but now the typography mixin does that. The resulting CSS is identical to what it is on master though 👍